src/Repository/CommitteeRepository.php

```diff
@@ -358,6 +359,48 @@ public function countApprovedForReferent(Adherent $referent): int
         ;
     }
 
+    public function findApprovedForReferentAutocomplete(Adherent $referent, $value): array
+    {
+        $this->checkReferent($referent);
+
+        $qb = $this->createQueryBuilder('committee')
+            ->select('committee.uuid, committee.name')
+            ->join('committee.referentTags', 'tag')
+            ->where('committee.status = :status')
+            ->andWhere('committee.name LIKE :searchedName')
+            ->andWhere('tag.id IN (:tags)')
+            ->setParameter('searchedName', $value.'%')
+            ->setParameter('status', Committee::APPROVED)
+            ->setParameter('tags', $referent->getManagedArea()->getTags())
+            ->orderBy('committee.name')
+        ;
+
+        return array_map(function (array $committee) {
```

Comment:
is it really needed?
what about return just array of :
```
[
  {
    "uuid": "uuid_1",
    "name": "name_1"
  },
  {
    "uuid": "uuid_2",
    "name": "name_2"
  }
]
```

instead of

```
[
  {
    "uuid_1": "name_1"
  },
  {
    "uuid_2": "nama_2"
  }
]
```

Reply:
It was defined with front dev.